modules/ROOT/pages/auth.adoc

```diff
@@ -0,0 +1,170 @@
+= Luna Streaming Authentication
+
+== Authentication
+
+The Helm chart can enable token-based authentication for your Pulsar cluster. For information on token-based
+authentication in Pulsar, go https://pulsar.apache.org/docs/en/security-token-admin/[here].
+
+For authentication to work, the token-generation keys need to be stored in Kubernetes secrets along with some default tokens (for superuser access). 
+
+The Helm chart includes tooling to automatically create the necessary secrets, or you can do this manually.
+
+=== Automatic generation of secrets for Pulsar token authentication
+
+Use the following settings to enable automatic generation of the secrets and enable token-based authentication:
+
+----
+enableTokenAuth: yes
+autoRecovery:
+  enableProvisionContainer: yes
+----
+
+When the provision container is enabled, it will check if the required secrets exist. If they don't exist, it will generate new token keys and use those keys to generate the default set of tokens. 
```

Comment:
How do I get from enabling > secrets > tokens? What do I do with the tokens next?